python_modules/libraries/dagster-k8s/dagster_k8s/executor.py

```diff
@@ -167,21 +169,30 @@ def __init__(
             batch_api_override=k8s_client_batch_api
         )
 
+    def _get_step_key(self, step_handler_context: StepHandlerContext) -> str:
+        step_keys_to_execute = cast(
+            List[str], step_handler_context.execute_step_args.step_keys_to_execute
+        )
+        assert len(step_keys_to_execute) == 1, "Launching multiple steps is not currently supported"
+        return step_keys_to_execute[0]
+
     def _get_container_context(self, step_handler_context: StepHandlerContext):
-        run_target = K8sContainerContext.create_for_run(
+        step_key = self._get_step_key(step_handler_context)
+
+        context = K8sContainerContext.create_for_run(
             step_handler_context.pipeline_run,
             cast(K8sRunLauncher, step_handler_context.instance.run_launcher),
+            include_run_tags=False,  # For now don't include job-level dagster-k8s/config tags in step pods
```

Comment:
I think I will probably just revert the k8s_job_op part when its safe to do breaking changes to experimental components - k8s_job_op is still experimental so there is a bit of wiggle room there

Reply:
from https://docs.dagster.io/getting-started/releases

> Experimental APIs may change or disappear within any release, but we try to avoid breaking them within minor releases if they have been around for a long time.

So I think you could change it now if you wanted. If you don't want to pull the trigger now maybe just stack the change PR and sit on it till the release you want to send it - just to ensure we don't lose track of it.